.github/workflows/build_docker_image.yml

```diff
@@ -39,18 +42,59 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Build and push
+      uses: docker/build-push-action@v6
+      with:
+        push: ${{ github.ref == 'refs/heads/main' }}
+        tags: ghcr.io/f3d-app/f3d-wasm:${{ env.VTK_SHA }}
+        context: "{{defaultContext}}:webassembly"
+        build-args: |
+          CMAKE_BUILD_PARALLEL_LEVEL=4
+          ASSIMP_VERSION=v5.4.2
+          DRACO_VERSION=1.5.7
+          OCCT_VERSION=V7_8_1
+          VTK_VERSION=${{ env.VTK_SHA }}
+
+  build_f3d-android:
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [armeabi-v7a, arm64-v8a, x86, x86_64]
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
```

Comment:
what do we need buildx for ?

Reply:
Ah, I forgot to remove it. We need it if we want to use caches that I plan to implement at some point.